master/internal/core.go

```diff
@@ -994,6 +994,18 @@ func (m *Master) Run(ctx context.Context) error {
 		default:
 			hasMatchingFile = !stat.IsDir()
 		}
+
+		// Files that receive a unique hash when bundled and deployed can be cached forever
+		// Other static files should only be cached for a short period of time
+		cacheFileLongTerm := regexp.MustCompile(`.(chunk\.(css|js)|woff2|woff)$`)
+		cacheFileShortTerm := regexp.MustCompile(`.(ico|png|jpe*g|gif|svg)$`)
+
+		if cacheFileLongTerm.MatchString(requestedFile) {
+			c.Response().Header().Set("cache-control", "max-age=31536000")
+		} else if cacheFileShortTerm.MatchString(requestedFile) {
+			c.Response().Header().Set("cache-control", "max-age=600")
```

Comment:
This seems somewhat "dangerous" to blanket set this across all responses since we run a highly interactive website. Also given the small userbase not much of a win.